Platform.md

```diff
@@ -76,6 +76,8 @@ This file contains instructions for installing dependencies on different platfor
 
   * Both the Websocket and OpenSSL builds work on windows. OpenSSL 1.0.2 needs to be installed for them to work properly.
   * Download and install OpenSSL from [this link](https://slproweb.com/products/Win32OpenSSL.html)
+    * Make sure you download the "full" ~30mb version, the "light" version won't work.
```

Comment:
It would help to elaborate on why the light version won't work (e.g..: The light version does not include the files necessary to build the SDK")